bin/crew

```diff
@@ -1509,25 +1540,29 @@ def upload(pkg_name = nil, pkg_version = nil)
       new_version = noname.split('-chromeos').first
       new_url = "#{base_url}/#{package}/#{new_version}_#{arch}/#{new_tarfile}".gsub("#{release_dir}/", '')
       token_label = gitlab_token.split('-').first == 'glpat' ? 'PRIVATE-TOKEN' : 'DEPLOY-TOKEN'
-      puts "curl -# --header \"#{token_label}: #{gitlab_token}\" --upload-file \"#{new_tarfile}\" \"#{new_url}\" | cat" if CREW_VERBOSE
-      output = `curl -# --header "#{token_label}: #{gitlab_token}" --upload-file "#{new_tarfile}" "#{new_url}" | cat`.chomp
-      if output.include?('201 Created')
-        puts "curl -Ls #{new_url} | sha256sum" if CREW_VERBOSE
-        upstream_sha256 = `curl -Ls #{new_url} | sha256sum`.chomp.split.first
-        if upstream_sha256 == new_sha256
-          puts output.lightgreen
-        else
-          if CREW_VERBOSE
-            puts "expected sha256 hash=#{new_sha256}"
-            puts "upstream sha256 hash=#{upstream_sha256}"
+      if `curl -sI #{new_url}`.lines.first.split[1] == '200'
+        abort "#{new_tarfile} has already been uploaded. Please change the #{package} package version from #{new_version} and try again.".lightred
```

Comment:
I don't think this is a good idea.  I've built binaries and found out later they had issues but there was no need to bump the version since the package changes were not committed yet.  It doesn't do any harm to overwrite existing binaries as long as the correct sha256 hashes accompany them in the package.